torch/distributed/checkpoint/default_planner.py

```diff
@@ -78,7 +79,12 @@ def __init__(
                 "from your call."
             )
 
-    def set_up_planner(self, state_dict: STATE_DICT_TYPE, is_coordinator: bool) -> None:
+    def set_up_planner(
+        self,
+        state_dict: STATE_DICT_TYPE,
+        storage_meta: Optional[StorageMeta],
```

Comment:
Use keywoard argument to avoid BC linter.

Reply:
Added defaults to kwargs, but I think the issue is I wanted to match the order of args from `set_up_planner` in the `LoadPlanner`. I believe this is dealt with in a BC supporting way using inspect at the callsite

Reply:
Since rank0 is actually the rank that writes the metadata file, the rank0 save_id will be the one that is saved.